go/vt/vtgate/evalengine/testcases/inputs.go

```diff
@@ -155,7 +155,6 @@ var inputStrings = []string{
 	"-9223372036854775808",
 	"999999999999999999999999",
 	"-999999999999999999999999",
-	"_latin1 X'ÂÄÌå'",
```

Comment:
Dropped this case since it's actually a syntax error and not really a useful test here.